website/docs/reference/typegate/index.mdx

```diff
@@ -46,3 +46,32 @@ GraphQL, being a query language, offers a great asset for Metatype's philosophy:
 2. Flexibility: allows for retrieving multiple resources in a single request, unlike REST, which often requires multiple ones
 3. Typing: GraphQL has a built-in type system that allows for better documentation and stronger validation of the requests
 4. Improved tooling: tools and libraries around GraphQL are rapidly growing and great a development experience
+
+
+## Configuration
+
+### Environment variables.
+
+
+| Environment variables     | Desc                                                             | Default                 |
+| ------------------------- | ---------------------------------------------------------------- | ----------------------- |
+| HOSTNAME                  | Hostname that typegate is deployed on.                           | `getHostname()` result. |
+| TG_PORT                   | Tcp port to serve typegate APIs at.                              | 7890                    |
+| TG_ADMIN_PASSWORD         | Password use by the CLI/SDK to configure the typegate.           | **Required**            |
+| TG_SECRET                 | Symmetric key used to encrypt cookies and other things.          | **Required**.           |
+| TMP_DIR                   | Top-level temporary directory.                                   | `$PWD/tmp`              |
+| DEBUG                     | Enable debug output and other development paths.                 | false                   |
+| TIMER_MAX_TIMEOUT_MS      | Timeout for custom runtime functions and other proccesses.       | 3000                    |
+| TIMER_DESTROY_RESOURCES   |                                                                  | true                    |
+| TIMER_POLICY_EVAL_RETRIES |                                                                  | 1                       |
+| JWT_MAX_DURATION_SEC      | The lifetime of generated JWT access tokens.                     | `30 * 24 * 3600`        |
+| JWT_REFRESH_DURATION_SEC  | The lifetime of generated JWT refresh tokens.                    | `5 * 60`                |
+| SENTRY_DSN                |                                                                  |                         |
+| SENTRY_SAMPLE_RATE        |                                                                  | 1                       |
+| SENTRY_TRACES_SAMPLE_RATE |                                                                  | 1                       |
+| TRUST_PROXY               | Weather to accept proxy headers when resolving request contexts. | false                   |
+| TRUST_HEADER_IP           | The header key on which to resolve request origin addresses.     | X-Forwarded-For         |
+| DENO_V8_FLAGS             | Flags for tuning the v8 javascript engine.                       |                         |
+
+
```

Comment:
We have more [here](https://github.com/metatypedev/metatype/blob/1c6c8afeb6bebc44b672ef61efae40a58689f8b2/website/docs/reference/typegate/synchronization/index.mdx?plain=1#L52).
Perhaps a link or re-add them?